helmfile/charts/log-manager/values.yaml

```diff
@@ -0,0 +1,57 @@
+fullnameOverride: ""
+nameOverride: ""
+
+imagePullSecrets: []
+image:
+  pullPolicy: IfNotPresent
+  repository: harbor.aarnq.dev-ck8s.com/ck8s/log-manager
```

Comment:
Point to a public repo with the image?

Reply:
Forgot about pushing that one :sweat_smile: